grails-app/controllers/org/pih/warehouse/api/PurchaseOrderApiController.groovy

```diff
@@ -82,7 +82,7 @@ class PurchaseOrderApiController {
 
     def statusOptions = {
         def options = OrderSummaryStatus.derivedStatuses()?.collect {
-            [ id: it.name(), value: it.name(), label: "${g.message(code: 'enum.OrderSummaryStatus.' + it.name())}", variant: it.variant ]
+            [ id: it.name(), value: it.name(), label: "${g.message(code: 'enum.OrderSummaryStatus.' + it.name())}", variant: "${g.message(code: 'enum.StatusType.' + it.variant)}"  ]
```

Comment:
This should not be translatable because it might not work properly on the frontend after other language translations are provided